core/contract/src/main/java/org/eclipse/dataspaceconnector/contract/negotiation/ConsumerContractNegotiationManagerImpl.java

```diff
@@ -146,19 +150,24 @@ public NegotiationResult offerReceived(ClaimToken token, String negotiationId, C
             //    negotiation.addContractOffer(result.getCounterOffer());
             //    monitor.debug("[Consumer] Contract offer received. A counter offer is available.");
             //    negotiation.transitionOffering();
+            //    negotiationStore.save(negotiation);
```

Comment:
If this commented out code is there because we miss something I would open an issue and delete that block.

Reply:
This code was a first attempt at introducing counter offers to the negotiation, but since we currently don't have a service or hook for creating counter offers, it was commented out. I'll remove the commented out code and check if there is already an issue for integrating counter offers. If not, I'll create one.